cobrawap/pipeline/stage04_wave_detection/scripts/wave_mode_clustering.py

```diff
@@ -351,11 +351,10 @@ def clean_timelag_dataframe(df, min_trigger_fraction=.5,
     for i, cluster_grid in enumerate(mode_grids):
         pattern_func = interpolate_grid(cluster_grid, args.interpolation_smoothing)
         fx, fy, pattern = sample_wave_pattern(pattern_func,
-                                            step=args.interpolation_step_size,
-                                            dim_x=dim_x, dim_y=dim_y)
-        interpolated_mode_grids = np.concatenate((interpolated_mode_grids,
-                                                    pattern[np.newaxis,:])) \
-                                        if i else pattern[np.newaxis,:]
+                                              step=args.interpolation_step_size,
+                                              dim_x=dim_x, dim_y=dim_y)
+        interpolated_mode_grids = np.concatenate((interpolated_mode_grids, pattern[np.newaxis,:])) \
+                                      if i else pattern[np.newaxis,:]
```

Comment:
This line is too long. I suggest the following formatting:

```suggestion
        interpolated_mode_grids = np.concatenate((interpolated_mode_grids,
                                                  pattern[np.newaxis,:])) \
                                  if i else pattern[np.newaxis,:]
```

Reply:
Ok, agreed. Fixed.